node/node_handler.go

```diff
@@ -397,17 +399,22 @@ func (node *Node) VerifyNewBlock(newBlock *types.Block) error {
 
 func (node *Node) numSignaturesIncludedInBlock(block *types.Block) uint32 {
 	count := uint32(0)
-	pubkeys := node.Consensus.Decider.Participants()
-	mask, err := internal_bls.NewMask(pubkeys, nil)
+	members := node.Consensus.Decider.Participants()
```

Comment:
from participants to mask could be cached per epoch. may be in the next PR.

Reply:
yes. it's a todo.